pyproject.toml

```diff
@@ -24,9 +24,10 @@ packages = [
 "ipfabric" = "nautobot_chatops_ipfabric.worker:ipfabric"
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.7.1"
 nautobot-chatops = "^1.1.0"
 netutils = "^1.0.0"
+ipfabric-diagrams = "^1.2.7"
```

Comment:
even `ipfabric` is a dependency from `ipfabric-diagrams`,  I believe it makes sense to track it in a separate item, to adjust as needed.

Reply:
so add `ipfabric= "^0"` as a dependency?

Reply:
Not required as it is a dependency of ipfabric-diagrams already. https://github.com/community-fabric/python-ipfabric-diagrams/blob/0e7beaab48c828aca8e249c0c9082827f96b2007/pyproject.toml#L25

Reply:
yes, I know it is pinned, but having as a separate one would led us to tune it if necessary and make it more explicit. Not a hard point, but I would vote for adding both.

Reply:
ok, we can add it in. My assumption would be that with diagrams we'll always need whatever `ipfabric-diagrams` needs but we can add it in for now.